app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ContactsController < ApplicationController
+  
+  def create
+    contact = Contact.new(contact_params)
+    contact.company_id = params[:company_id]
+    contact.save
```

Comment:
what happens if our contact doesn't save? a user should probably see a nice error message and not that they added the contact!